webClient/src/app/core/menu-bar/menu-bar.component.ts

```diff
@@ -231,6 +246,19 @@ export class MenuBarComponent implements OnInit {
     */
   }
 
+  deleteFile() {
+    let deleteFileRef = this.dialog.open(DeleteFileComponent, {
+      width: '500px'
+    });
+
+    deleteFileRef.afterClosed().subscribe(result => {
+      if (result) {
+        console.log("Deleting: " + result);
```

Comment:
Try to `@Inject(Angular2InjectionTokens.LOGGER) private log: ZLUX.ComponentLogger` and then use that logger.